### DISCUSSION TITLE
toogle dts/ac3 bitstream on/off via key binding

### DISCUSSION THREAD

**Discussion by AlexD81** [mpv-player/mpv#15065] [Q&A]
hi, how can i archieve it in android ? 
when i press specific key, audio device has to change, and bitstream  get enabled. On second press the same key, it has to revert back. How can i do it ?
commands for bitstream are looking similar to this, i suppose.

audio-device='alsa/iec958:CARD=Audio,DEV=0'
audio-spdif=ac3,dts

looking for help, pls

**Comment by guidocella** [comment]
https://mpv.io/manual/master/#command-interface-cycle-values

**Reply by AlexD81** [reply]
@guidocella   thanks for advance,     need more help pls,   it's too difficult to manage it myself (     
 how can i  get  audio-device  name in android?    and then, what to write in cycle values, and how to create key bindings to toggle...

**Comment by Nojevah** [comment]
It looks like toggling bitstreaming is not possible. It can only be enabled/disabled in config file (or command line).
Tried unsuccessfully with profile-cond or cycle-values audio-spdif "" "ac3,dts,dts-hd,truehd" on Windows 11.
